htdocs/accountancy/class/accountingjournal.class.php

```diff
@@ -103,6 +104,50 @@ public function __construct($db)
 		$this->ismultientitymanaged = 0;
 	}
 
+	/**
+	 * Create a new Accounting Journal.
+	 *
+	 * @param  User	$user the user that created the journal, currently unused
+	 * @return int  Return integer <0 on error, or the ID of the created object
+	 */
+	public function create($user)
+	{
+		$valid_nature = array(1, 2, 3, 4, 5, 8, 9);
+		if (!in_array((int) $this->nature, $valid_nature)) {
+			$this->error = get_class($this)."::Create Error invalid field nature '" . strval($this->nature) . "'";
+			dol_syslog($this->error, LOG_ERR);
+			return -1;
+		}
+
+		$sql = "INSERT INTO ".MAIN_DB_PREFIX."accounting_journal";
+		$sql .= " (entity, code, label, nature, active)";
+		$sql .= " VALUES ("
+			. ((int) $this->entity)					 .",'"
+			. $this->db->escape($this->code)          ."','"
+			. $this->db->escape($this->label)         ."',"
+			. ((int) $this->nature)					 .","
+			. ((int) $this->active)                    .")";
+
+		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		$resql = $this->db->query($sql);
+		if (!$resql) {
+			$this->error = get_class($this)."::Create Error " . $resql . ": " . $this->db->lasterror();
```

Comment:
Remove the $resql frim the log.

Reply:
Done

Reply:
mmh, not sure why I got the changes from the other MR there with a rebase origin/develop, but sorry for that.